src/particles.h

```diff
@@ -71,9 +322,20 @@ struct ParticleParameters : CommonParticleParams {
 
 struct ParticleSpawnerParameters : CommonParticleParams {
 	u16 amount = 1;
-	v3f minpos, maxpos, minvel, maxvel, minacc, maxacc;
 	f32 time = 1;
-	f32 minexptime = 1, maxexptime = 1, minsize = 1, maxsize = 1;
+
+	// texture is used for the fallback texture on outdated clients
+
+	std::vector<ServerParticleTexture> texpool;
+
+	ParticleParamTypes :: v3fRangeTween
```

Comment:
no spaces around the double dots

Reply:
@nerzhul this was fixed months ago and was trivial in any case, please approve